test/xml/CatLibrary/Cat`2.xml

```diff
@@ -112,14 +112,14 @@
     <altmember cref="T:CatLibrary.ICat" />
     <example>
       <para>Here's example of how to create an instance of this class. As T is limited with <c>class</c> and K is limited with <c>struct</c>.</para>
-      <code language="c#"><![CDATA[
-                var a = new Cat(object, int)("&lt;");
+      <code language="c#">
```

Comment:
what's the point of changing here?

Reply:
I didn't modify this. It brought this change by itself when executing the command.
I don't need to modify the code, then to execute  .\gen-xml.ps1 And there will be this change.